### PR TITLE
Luke

### PR DESCRIPTION
**Pull request by u19010240**
Finalizing Venue CRUD:

- Made sure Global service is used for:
- alerts
- toasts
- modals

Removed unused imports and ensured delete venue checks for associated schedule records - if any exist it shows the associative component. 

Fixed naming consistency in headers and all CRUDVA modals for venue. 

Moved Repository folder out of models folder in the API - but kept the namespace the same so no dependencies are broken.

### CODE REVIEW

File: Core API/Team7/.vs/Team7/config/applicationhost.config [hunk 1]
```diff
@@ -157,7 +157,7 @@
       </site>
       <site name="Team7" id="2">
         <application path="/" applicationPool="Team7 AppPool">
-          <virtualDirectory path="/" physicalPath="D:\Users\shann\Desktop\MasterFinal\HIIT-Team7\Core API\Team7" />
```
Comment (u19010240) [inline Core API/Team7/.vs/Team7/config/applicationhost.config:160]: This should not be changing on developer

File: Core API/Team7/Team7.csproj [hunk 2]
```diff
@@ -5,7 +5,7 @@
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
-    <OutputPath>D:\Users\shann\Desktop\New folder (2)\</OutputPath>
+    <OutputPath></OutputPath>
```
Comment (u19010240) [inline Core API/Team7/Team7.csproj:8]: Removing this prevents it breaking for anyone

File: Ionic/src/app/app-routing.module.ts [hunk 3]
```diff
@@ -9,7 +10,8 @@ const routes: Routes = [
   },
   {
     path: 'home',
-    loadChildren: () => import('./pages/home/home.module').then( m => m.HomePageModule)
+    loadChildren: () => import('./pages/home/home.module').then( m => m.HomePageModule),
+    //canActivate: [AuthGuard] //
```
Comment (u19010240) [inline Ionic/src/app/app-routing.module.ts:14]: This is how we will be able to Guard routes in the sideMenu component

File: Ionic/src/app/pages/venue/view-venue-info/view-venue-info.component.html [hunk 4]
```diff
@@ -2,44 +2,44 @@
...
 <ion-content>
   <br/>
   <div class="ion-text-center">
-    <p style="font-size:large; color: white;margin-left: 10%;margin-right: 10%;">The following information is associated with the <br/> "{{venue.name}}" venue</p>
+    <p style="font-size:large; color: white;margin-left: 10%;margin-right: 10%;">The following information is associated with the venue.</p>
   
 
     <ion-item > 
       <ion-label>
-        <h2>Venue Name:</h2>
```
Comment (u19010240) [inline Ionic/src/app/pages/venue/view-venue-info/view-venue-info.component.html:18]: We need to make sure this is consistent for all CRUDS + Documentation

**Review by u19010240 — COMMENTED** [review]
Initial review -> 86 files changed when moving the repository folder out of the models folder in the API, so I clicked view on all those to allow us to filter them out when going over the changes to venue CRUD.

File: Core API/Team7/.vs/Team7/config/applicationhost.config (hunk 1 above)
Comment (besteeezy) [inline Core API/Team7/.vs/Team7/config/applicationhost.config:160]: Adding Shan's as we always use hers for demo

File: Core API/Team7/Team7.csproj (hunk 2 above)
Comment (besteeezy) [inline Core API/Team7/Team7.csproj:8]: Leaving it empty as it will stop the breaking.

**Review by besteeezy — APPROVED** [review]

**Review by GeorgeLiatos — APPROVED** [review]

**Review by EllenLyne — APPROVED** [review]

**Pull request merged**